packages/account/src/providers/provider.ts

```diff
@@ -739,12 +740,15 @@ Supported fuel-core version: ${supportedVersion}.`
    * @param sendTransactionParams - The provider send transaction parameters (optional).
    * @returns A promise that resolves to the transaction response object.
    */
-  // #region Provider-sendTransaction
   async sendTransaction(
     transactionRequestLike: TransactionRequestLike,
     { estimateTxDependencies = true }: ProviderSendTxParams = {}
   ): Promise<TransactionResponse> {
     const transactionRequest = transactionRequestify(transactionRequestLike);
+    if (isTransactionTypeUnknown(transactionRequest)) {
+      return new TransactionResponse(transactionRequest, this);
+    }
+    // #region Provider-sendTransaction
```

Comment:
@maschad My understanding is that if someone attempts to submit a transaction that the SDK does not support, we should throw an error rather than simply resolving the `sendTransaction` promise with a dummy response.

Reply:
@Torres-ssf If a consumer calls [`batchTransfer`](https://github.com/FuelLabs/fuels-ts/blob/mc/chore/handle-unknown-txns/packages/account/src/account.ts#L371) (which calls `sendTransaction`) for 10 transactions and 1 of them is currently unsupported by the SDK version should all transactions not be submitted and an error thrown?